src/index.html

```diff
@@ -4,9 +4,387 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home" class="logo">
+          <img
+            src="./images/logo.png"
+            alt="logo"
+            class="logo__nav"
+          >
+        </a>
+      </div>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item"><a href="#" class="nav__link">Samsung</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Smartphones</a></li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav__item"><a href="#" class="nav__link">Gadgets</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Tablets</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Photo</a></li>
+          <li class="nav__item"><a href="#" class="nav__link">Video</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="product-card" data-qa="card">
+
+        <div class="product-card__item">
```

Comment:
```suggestion
      <div class="product-card" data-qa="card">
        <div class="product-card__item">
```
no need to add blank between parent and the first child